packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -519,6 +530,14 @@ export const workflowsRouter = router({
           });
           //step was edited
         } else if (JSON.stringify(oldStep) !== JSON.stringify(newStep)) {
+          if (
+            !userWorkflow.user.teams.length &&
+            !isSMSAction(oldStep.action) &&
```

Comment:
We still allow to edited already existing SMS workflow actions for users that don't belong to a team